config/environments/production.rb

```diff
@@ -81,7 +81,11 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
-  config.logstasher.enabled = true
-  config.logstasher.logger = Logger.new("#{Rails.root}/log/#{Rails.env}.json.log")
-  config.logstasher.suppress_app_log = true
+  if ENV["DYNO"].present?
```

Comment:
If this app used 12 factor logging [like Publishing API](https://github.com/alphagov/publishing-api/blob/master/config/environments/production.rb#L53-L57), this conditional shouldn't be necessary. It'd be good to move to this approach anyway, as per [the accepted RFC](https://gov-uk.atlassian.net/wiki/display/TECH/RFC+26%3A+12-Factor+Rails+Apps).